src/resolvers/__tests__/collectionDraft.test.js

```diff
@@ -28,9 +28,11 @@ describe('Collection', () => {
             .defaultReplyHeaders({
               'X-Request-Id': 'abcd-1234-efgh-5678'
             })
-            .get(/collection_drafts/)
+            .get(/api/)
```

Comment:
shouldn't the endpoint be /api/drafts now?

Reply:
Updated to `/api\/drafts/`.